indicators/models.py

```diff
@@ -1285,8 +1285,6 @@ class Indicator(SafeDeleteModel):
         _("Edit date"), null=True, blank=True, help_text=" "
     )
 
-    notes = models.TextField(_("Notes"), max_length=500, null=True, blank=True)
```

Comment:
This may have already been addressed in the ticket, but the char length on the notes field is 500, while the length on the comments field is 225.  Maybe we need to increase length of comment field?